packages/unenv-preset/src/runtime/node/module.ts

```diff
@@ -53,17 +53,20 @@ export {
 	runMain,
 	syncBuiltinESMExports,
 	wrap,
-} from "unenv/runtime/node/module/index";
+} from "unenv/node/module";
 
 const workerdModule = process.getBuiltinModule("node:module");
 
 export const createRequire: typeof nodeModule.createRequire = (
 	file: string | URL
 ) => {
 	return Object.assign(workerdModule.createRequire(file), {
-		resolve: Object.assign(notImplemented("module.require.resolve"), {
-			paths: notImplemented("module.require.resolve.paths"),
-		}),
+		resolve: Object.assign(
+			/*@__PURE__*/ notImplemented("module.require.resolve"),
+			{
+				paths: /*@__PURE__*/ notImplemented("module.require.resolve.paths"),
```

Comment:
The idea, here, being that by marking these as PURE the bundlers can remove these if not used?

Reply:
That's exact.
This is reflecting the many `/*@__PURE__*/` annotation added by @pi0 in unenv@2

Reply:
I think it won't affect anything for bundlers when used _inside_ functions (might be accident if I also added it in unenv). PURE is for module top level context.

Reply:
At least is will not harm, let's get the PR in and we can investigate later